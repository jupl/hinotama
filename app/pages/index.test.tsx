import {shallow} from 'enzyme'
import * as React from 'react'
import Index from '.'

describe('/ page', () => {
  it('should render as expected', () => {
    expect(shallow(<Index isServer />)).toMatchSnapshot()
    expect(shallow(<Index isServer={false} />)).toMatchSnapshot()
  })
})
