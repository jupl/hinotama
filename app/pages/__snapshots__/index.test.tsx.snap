// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/ page should render as expected 1`] = `
<styled.div>
  Hello, World
</styled.div>
`;
